modules/rodoo-update-active-deadline-seconds.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/run_once_duration_override/run-once-duration-override-install.adoc
+
+:_content-type: PROCEDURE
+[id="rodoo-update-active-deadline-seconds_{context}"]
+= Updating the run-once active deadline override value
+
+You can customize the override value that the {run-once-operator} applies to run-once pods. The default value is `3600` seconds, or 1 hour.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have installed the {run-once-operator}.
+
+.Procedure
+
+. Log in to the OpenShift CLI.
+
+. Edit the `RunOnceDurationOverride` resource:
++
+[source,terminal]
+----
+$ oc edit runoncedurationoverride cluster
+----
+
+. Update the `activeDeadlineSeconds` field:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: RunOnceDurationOverride
+metadata:
+# ...
+spec:
+  runOnceDurationOverride:
+    spec:
+      activeDeadlineSeconds: 1800 <1>
+# ...
+----
+<1> Set the `activeDeadlineSeconds` field to the desired value, in seconds.
+
+. Save the file to apply the changes.
+
+// TODO: Confirm that this value take effect only for new pods or when pods are updated
```

Comment:
@ingvagabund Is the behavior the same when updating the override value as when you first label the namespace? Like only newly created pods in the ns or updated pods would get the new value, and all of the existing ones would still have the old value?

Reply:
Exactly.